internal/pkg/service/buffer/storage/local/writer/count/counter.go

```diff
@@ -0,0 +1,103 @@
+// Package count provides a counter with backup to a file.
+package count
+
+import (
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/strhelper"
+	"go.uber.org/atomic"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+)
+
+// Counter is an atomic counter.
+type Counter struct {
+	value *atomic.Uint64
+}
```

Comment:
Skopiroval som `size.Meter` -> `count.Counter` - je to pocitadlo, ktore si zalohuje hodnotu na disk.
`size.Meter` mal `Write` metodu, `count.Counter` ma `Add` metodu.

Reply:
![image](https://github.com/keboola/keboola-as-code/assets/19371734/7e354f4a-a286-4367-ba01-45a8f0322371)